.github/workflows/release.yml

```diff
@@ -125,16 +117,35 @@ jobs:
           cp ${{ matrix.build.production_target }}/subspace-cli ${{ matrix.build.production_target }}/macos-binaries
           ditto -c -k --rsrc ${{ matrix.build.production_target }}/macos-binaries subspace-binaries.zip
           echo "Notarizing"
-          xcrun altool --notarize-app --primary-bundle-id binaries-${{ github.ref_name }} --username "${{ secrets.MACOS_APPLE_ID}}" --password "${{ secrets.MACOS_APP_PASSWORD }}" --file subspace-binaries.zip
-          # echo "Stapling farmer"
-          # xcrun stapler staple ${{ matrix.build.production_target }}/subspace-cli
+          curl -sL https://github.com/mitchellh/gon/releases/download/v0.2.5/gon_macos.zip -o gon.zip
+          unzip gon.zip -d .
```

Comment:
Why not installing using homebrew like recommended in readme?

Reply:
It said the "easiest way" was to use brew, but if you prefer that method I can do that.

Reply:
Yeah, I think I'd prefer brew to manually downloading things with curl